Dockerfile

```diff
@@ -1,13 +1,32 @@
-FROM python:3.9
+FROM python:3.9-slim
 
 WORKDIR /code
 
+RUN \
+	apt update \
+	&& apt install -y \
+	# GDAL headers are required for fiona, which is required for geopandas.
+	# Also gcc is used to compile C++ code.
+	libgdal-dev g++ \
+	# libspatialindex is required for rtree.
+	libspatialindex-dev \
+	# Remove package index obtained by `apt update`.
+	&& rm -rf /var/lib/apt/lists/*
+
 COPY ./requirements.txt /code/requirements.txt
 
-RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+RUN \
+	pip install --no-cache-dir --upgrade -r /code/requirements.txt \
+	# For some reason, --no-cache-dir doesn't work. :(
+	&& rm -rf ~/.cache/pip
+
+ENV RIDE2GO_TOKEN=''
```

Comment:
why set it if it's empty?

Reply:
To make it explicit that it can be provided when running the container. We can remove it if you want.